pkgs/development/python-modules/pytest-responses/default.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytest,
+  pytestCheckHook,
+  responses,
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-responses";
+  version = "0.5.1";
+  format = "setuptools";
```

```suggestion
  pyproject  =true;
```

and add `build-system = [ setuptools ];`.